scripts/build/bundler.js

```diff
@@ -249,6 +251,18 @@ function getWebpackConfig(bundle) {
       // https://github.com/webpack/webpack/issues/6642
       globalObject: 'new Function("return this")()',
     },
+    resolve: {
+      fallback: {
+        os: false,
+        path: false,
+        util: false,
+        url: false,
+        // Webpack@5 can't resolve them
+        // Possible related issue https://github.com/webpack/webpack/issues/11467
+        "postcss/lib/parser": require.resolve("postcss/lib/parser"),
+        "postcss/lib/stringifier": require.resolve("postcss/lib/stringifier"),
```

Comment:
This is not good, it makes the `require('postcss')` in `less` parser resolves to `postcss@8`.

Not sure how to make it work.